tests/unit/nupic/algorithms/anomaly_likelihood_test.py

```diff
@@ -610,16 +654,185 @@ def testEstimationWindowSize(self):
     self.assertEqual(len(l._historicalScores), 3)
 
 
+  def testAnomalyLikelihoodWindowSizeImpactOnDistributionEstimateScoreArgs(
```

Comment:
These tests are in the class `AnomalyLikelihoodTest` so I think you can leave "AnomalyLikelihood" out of the names.